Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/LearnWizard/POMGeneralDetailsWizardPage.xaml.cs

```diff
@@ -69,10 +69,17 @@ public void WizardEvent(WizardEventArgs WizardEventArgs)
                     {
                         xTakeScreenShotLoadButton.Visibility = Visibility.Visible;
                     }
+                    SetDefaultPage();                    
                     break;
             }
         }
 
+        private void SetDefaultPage()
+        {
+            xRDBPageURL.IsChecked = true;
+            xBusinessFlowControl.TargetApplication = mWizard.mPomLearnUtils.POM.TargetApplicationKey.ItemName;
```

Comment:
why not to pass the GUID of the target application for more robust solution

Reply:
on the POM page also it is implemented on "ItemName" not on GUID, there can be scenario where if any target application is removed and again added with same details then the items with existing GUID gets filtered in the open window, so it is implemented in this way please let me know if need to implement on GUID